records.go

```diff
@@ -165,19 +165,30 @@ const (
 	RRTypeWKS RRType = "WKS"
 )
 
+// Define a function to create a comment option
+func WithComments(comments ...Comment) func(*RRset) {
```

Comment:
Can I ask you to add a small unit test for this function?

Reply:
I new test has been for this  function